### PR TITLE
Fixes #19622: Configure relayd to check certificates

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/19622

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Review by peckpeck — APPROVED** [review]

File: techniques/system/rudder-service-relayd/1.0/common/relayd.conf.tpl [hunk 1]
```diff
@@ -48,21 +54,15 @@ retention = "1day"
...
-url = "{{{vars.rudder_postgresql.db_url}}}"
-password = "{{{vars.rudder_postgresql.db_password}}}"
+url = "{{{vars.system_rudder_relay_configuration.db_url}}}"
+password = "{{{vars.system_rudder_relay_configuration.db_password}}}"
 {{/classes.root_server}}
 max_pool_size = 10
 
 [output.upstream]
-url = "https://{{{vars.server_info.policy_server}}}"
+host = "{{{vars.server_info.policy_server}}}"
```
Comment (ncharles) [inline techniques/system/rudder-service-relayd/1.0/common/relayd.conf.tpl:63]: why is it changing ?

**Pull request merged**